packages/xchain-ethereum/src/utils.ts

```diff
@@ -41,12 +42,52 @@ export const ethNetworkToXchains = (network: EthNetwork): XChainNetwork => {
   }
 }
 
-export const getTxFromOperation = (operation: TransactionOperation): Tx => {
+/**
+ * Validate the given address.
+ *
+ * @param {Address} address
+ * @returns {boolean} `true` or `false`
+ */
+export const validateAddress = (address: Address): boolean => {
+  try {
+    ethers.utils.getAddress(address)
+    return true
+  } catch (error) {
+    return false
+  }
+}
+
+/**
+ * Get token address from asset.
+ *
+ * @param {Asset} asset
+ * @returns {string|null} The token address.
+ */
+export const getTokenAddress = (asset: Asset): string | null => {
+  const assetAddress = asset.symbol.slice(asset.ticker.length + 1)
+  if (!validateAddress(assetAddress)) {
+    return null
+  }
+  return assetAddress
+}
```

Comment:
Since it's critical to get a valid address from `Asset`, write a test for `getTokenAddress` pls.